docs/input-kafka.asciidoc

```diff
@@ -528,6 +532,39 @@ http://kafka.apache.org/documentation.html#security_sasl[SASL mechanism] used fo
 This may be any mechanism for which a security provider is available.
 GSSAPI is the default mechanism.
 
+[id="plugins-{type}s-{plugin}-schema_registry_key"]
+===== `schema_registry_key`
+
+* Value type is <<string,string>>
+* There is no default value for this setting.
+
+Set the username for basic authorization to access remote Schema Registry.
+
+[id="plugins-{type}s-{plugin}-schema_registry_proxy"]
+===== `schema_registry_proxy`
+
+* Value type is <<uri,uri>>
+* There is no default value for this setting.
+
+Set the address of a forward HTTP proxy. An empty string is treated as if proxy was not set.
+
+[id="plugins-{type}s-{plugin}-schema_registry_secret"]
+===== `schema_registry_secret`
+
+* Value type is <<string,string>>
+* There is no default value for this setting.
+
+Set the password for basic authorization to access remote Schema Registry.
+
+[id="plugins-{type}s-{plugin}-schema_registry_url"]
+===== `schema_registry_url`
+
+* Value type is <<uri,uri>>
+
+This is the URI that points to an instance of https://docs.confluent.io/current/schema-registry/index.html[Schema Registry] service,
+used to manage Avro schemas. When the Schema Registry is configured it's avoided to specify also the <<plugins-{type}s-{plugin}-value_deserializer_class>>.
+On the Schema Registry service must be previously uploaded the Avro schemas that are used to deserialize the data from the specified topics.
+Those schemas must follow a naming convention with the pattern `<topic name>-value`.
```

Comment:
Please consider this rewording: 

The URI that points to an instance of the
https://docs.confluent.io/current/schema-registry/index.html[Schema Registry]
service, used to manage Avro schemas. 
Be sure that the Avro schemas for deserializing the data from the specified
topics have been uploaded to the Schema Registry service. 
The schemas must follow a naming convention with the pattern `<topic
name>-value`.

Use either the Schema Registry config option or the
<<plugins-{type}s-{plugin}-value_deserializer_class>> config option, but not
both.

Reply:
Please make sure that there is a return between the last line of this description and the next options anchor.